shared/src/main/scala/crystal/Pot.scala

```diff
@@ -31,18 +37,16 @@ sealed trait Pot[+A] {
   def flatMap[B](f: A => Pot[B]): Pot[B] =
     map(f).flatten
 
-  def toOption: Option[A] =
-    this match {
-      case Ready(a) => a.some
-      case _        => none
-    }
+  def toOption: Option[A] = fold(_ => none, _ => none, _.some)
 
   def toTryOption: Option[Try[A]] =
     this match {
       case Pending(_) => none
       case Error(t)   => Failure(t).some
       case Ready(a)   => Success(a).some
     }
+
+  def zoom[B](lens: A => B): Pot[B] = map(lens)
```

Comment:
a proper `Lens` cannot be used as we defined `Pot[+A]` and lens is `Lens[A, B]`

Reply:
Is this an alias for `map`? Why do we need it?

Reply:
I guess you're right. I was thinking to compose it with a lens but I don't think we can